argo/base/deployment.yaml

```diff
@@ -32,7 +32,9 @@ spec:
               apiVersion: v1
               fieldPath: metadata.namespace
         - name: IN_CLUSTER
-          value: "true"
+          value: 'true'
+        - name: ENABLE_WEB_CONSOLE
```

Comment:
What is this force? And how does it relate to finalizers?

Reply:
For me this is the way to enable Argo UI, if I want to see Argo version. See Argo deployment